src/main/java/org/gridsuite/actions/server/ContingencyListController.java

```diff
@@ -116,22 +116,22 @@ public ContingencyListController(ContingencyListService service) {
         }
     }
 
-    @PostMapping(value = "/filters-contingency-lists", consumes = MediaType.APPLICATION_JSON_VALUE)
-    @Operation(summary = "Create a filters contingency list")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The filters contingency list have been created successfully")})
-    public ResponseEntity<FiltersContingencyList> createFilterContingencyList(@RequestParam(required = false, value = "id") UUID id,
-                                                                              @RequestBody(required = true) FiltersContingencyListAttributes filtersContingencyListAttributes) {
+    @PostMapping(value = "/form-contingency-lists", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Create a form contingency list")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The form contingency list have been created successfully")})
+    public ResponseEntity<FormContingencyList> createFormContingencyList(@RequestParam(required = false, value = "id") UUID id,
+                                                                           @RequestBody(required = true) FormContingencyList formContingencyList) {
         return ResponseEntity.ok()
             .contentType(MediaType.APPLICATION_JSON)
-            .body(service.createFilterContingencyList(id, filtersContingencyListAttributes));
+            .body(service.createFormContingencyList(id, formContingencyList));
     }
 
-    @PutMapping(value = "/filters-contingency-lists/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
-    @Operation(summary = "Modify a filters contingency list")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The filters contingency list have been modified successfully")})
-    public ResponseEntity<Void> modifyFilterContingencyList(@PathVariable UUID id, @RequestBody(required = true) FiltersContingencyListAttributes filtersContingencyListAttributes) {
+    @PutMapping(value = "/form-contingency-lists/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Modify a form contingency list")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The form contingency list have been modified successfully")})
+    public ResponseEntity<Void> modifyFormContingencyList(@PathVariable UUID id, @RequestBody(required = true) FormContingencyList formContingencyList) {
```

Comment:
Not sure what is best here, should we keep the attributes class or not... 
@niconoir what do you think ?

Reply:
I'm not sure it is necessary to keep attributes class here. For script contingency lists creations or modifications, we use a ScriptContingencyList class, so it seems consistent to use FormContingencyList class for form contingency lists creations or modifications too.